mteb/models/jina_clip.py

```diff
@@ -129,31 +129,31 @@ def encode(  # type: ignore
             kwargs.pop("prompt_name")
         return self.model.encode_text(sentences, batch_size=batch_size, **kwargs)
 
-    def encode_queries(self, queries: list[str], batch_size: int = 32, **kwargs: Any):
-        if "prompt_name" in kwargs:
-            kwargs.pop("prompt_name")
-        sentences = [
-            "Represent this sentence for searching relevant passages: " + sentence
-            for sentence in queries
-        ]
-        emb = self.encode(
-            sentences, batch_size=batch_size, normalize_embeddings=True, **kwargs
-        )
-        return emb
-
-    def encode_corpus(
-        self,
-        corpus: list[dict[str, str]] | dict[str, list[str]],
-        batch_size: int = 32,
-        **kwargs: Any,
-    ):
-        if "prompt_name" in kwargs:
-            kwargs.pop("prompt_name")
-        sentences = corpus_to_texts(corpus)
-        emb = self.encode(
-            sentences, batch_size=batch_size, normalize_embeddings=True, **kwargs
-        )
-        return emb
+    # def encode_queries(self, queries: list[str], batch_size: int = 32, **kwargs: Any):
+    #     if "prompt_name" in kwargs:
+    #         kwargs.pop("prompt_name")
+    #     sentences = [
+    #         "Represent this sentence for searching relevant passages: " + sentence
```

Comment:
I think we just need to move this logic to be dependent of PromptType in the `encode` method like in [here](https://github.com/embeddings-benchmark/mteb/blob/5c38eb8530abc0c251e244fccbf74872bfabc2a1/mteb/models/nomic_models.py#L47). Will add that in this PR.

Reply:
Actually, that's no longer needed, and this model is supported by Sentence Transformers. I will use that instead. [[ref](https://huggingface.co/jinaai/jina-clip-v1)]